src/main/java/com/google/devtools/build/lib/rules/cpp/CppConfiguration.java

```diff
@@ -933,7 +933,7 @@ public boolean getExperimentalCppCompileResourcesEstimation() {
 
   @Override
   public boolean macosSetInstallName() {
```

Comment:
seems like we have to keep this for a bit otherwise everyone will break until they update rules_cc https://github.com/bazelbuild/rules_cc/pull/379 / apple_support https://github.com/bazelbuild/apple_support/pull/378